src/schema/nmdc.yaml

```diff
@@ -356,6 +357,10 @@ classes:
       doi:
         description: >-
           The dataset citation for this study
+      ess dive datasets:
```

Comment:
this should go under the top level "slots"

Reply:
moved this from the slot usage section on study to the top level slots section